packages/terra-application/src/application-navigation/private/common/_PopupMenuUser.jsx

```diff
@@ -32,6 +42,17 @@ const PopupMenuUser = ({ userConfig }) => {
       <div className={cx('info-container')}>
         <div aria-hidden className={cx('name')}>{userConfig.name}</div>
         {userConfig.detail ? <div className={cx('detail')}>{userConfig.detail}</div> : null}
+        { userActionConfig && (
```

Comment:
should we include additional check for callback and text prop as well ?? to prevent button from getting rendered when only text is provided and callback is not provided.
```suggestion
        { userActionConfig && (userActionConfig.userActionCallback && userActionConfig.text ) (
```

Reply:
Leaving it as it is. As discussed [above](https://github.com/cerner/terra-application/pull/305#discussion_r956255536)